service/controller/resource/deployment/template/template.go

```diff
@@ -0,0 +1,30 @@
+package template
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/giantswarm/microerror"
+	"github.com/markbates/pkger"
+)
+
+// GetARMTemplate returns the ARM template reading a json file locally using pkger.
+func GetARMTemplate() (map[string]interface{}, error) {
+	contents := make(map[string]interface{})
+
+	f, err := pkger.Open("/service/controller/resource/deployment/template/main.json")
+	if err != nil {
+		return contents, microerror.Mask(err)
+	}
+	defer f.Close()
+
+	b, err := ioutil.ReadAll(f)
+	if err != nil {
+		return contents, microerror.Mask(err)
+	}
+
+	if err := json.Unmarshal(b, &contents); err != nil {
+		return nil, err
+	}
```

Comment:
Not super important, but in general it's nice to do stuff by streaming when original interface is streaming. Therefore this would be roughly as follows:
``` go
    d := json.NewDecoder(f)
    if err := d.Decode(&contents); err != nil {
        return nil, microerror.Mask(err)
    }
```

**NOTE**: Masking missing for unmarshalling error.

Reply:
Please, take a look now.